cmd/opentelemetry/app/processor/resourceprocessor/resource_processor.go

```diff
@@ -15,29 +15,31 @@
 package resourceprocessor
```

Comment:
Clever reuse of the otel resource processor.  As far as I can tell this is being configured to support agent tags?

Reply:
To inject tags passed as flags or env property. The OTEL collector does not support flags or env vars at the moment.